test/e2e/page-objects/pages/dialog/select-network.ts

```diff
@@ -187,16 +191,12 @@ class SelectNetwork {
 
   async clickDiscoverButton(): Promise<void> {
     console.log('Click Discover button in network options');
-    await this.driver.clickElement(
-      '[data-testid="network-list-item-options-discover"]',
-    );
+    await this.driver.clickElement(this.discoverButton);
```

Comment:
in page object mocal, we should define selector as class variable to be reusable